internal/indexer/helpers_test.go

```diff
@@ -44,7 +46,16 @@ func getTestPackages(t *testing.T) []*packages.Package {
 // the the object with the matching name and the package that contains it.
 func findDefinitionByName(t *testing.T, packages []*packages.Package, name string) (*packages.Package, types.Object) {
 	for _, p := range packages {
-		for _, def := range p.TypesInfo.Defs {
+		idents := make([]*ast.Ident, 0, len(p.TypesInfo.Defs))
+		for ident := range p.TypesInfo.Defs {
+			idents = append(idents, ident)
+		}
+		sort.Slice(idents, func(i, j int) bool {
+			return idents[i].Pos() < idents[j].Pos()
+		})
```

Comment:
Whats the sort needed for?

Reply:
Stupid tests look at names but anonymous fields make things conflict. This sorts it so that we deterministically fetch the first one we encounter lexically in tests.